src/index.html

```diff
@@ -1,14 +1,48 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <header class="main-head">
+      <a href="#" class="logo">
+        <img src="./images/logo.png" alt="Logo-Moyo" />
+      </a>
+      <nav class="main-nav">
+        <ul class="list">
+          <li class="main-nav_items">
+            <a href="#" class="main-nav-links is-active">Apple</a>
```

Comment:
почему множественное число? ссылка и элемент навигации то один в своем виде
```suggestion
          <li class="main-nav_item">
            <a href="#" class="main-nav-link is-active">Apple</a>
```